### DISCUSSION TITLE
[3.x]: How can i change setpassword.html template?

### DISCUSSION THREAD

**Discussion by ManiakMill** [craftcms/cms#11510] [Q&A] [answered]
### Description
Hello! I need to change ..vendor\craftcms\cms\src\templates\setpassword.html but save the route, only add some visual elements etc. But nothing happened.

### Steps to reproduce

1. Create a folder profile and add a new custom file profile-setpassword.html;
2. Add to general config variable 'setPasswordPath'  => 'profile/profile-setpassword';
3. Still see the standard form from  ..vendor\craftcms\cms\src\templates\setpassword.html.

### Craft CMS version

3.7.45.2

### PHP version

8.1.5

### Operating system and version

Windows 11

### Database type and version

MySql 5.7.14

### Image driver and version

GD 8.1.5

### Installed plugins and versions

"aelvan/inlin": "^2.1",
"carlcs/craft-redactorcustomstyles": "3.0.4",
"craftcms/cms": "3.7.45.2",
"craftcms/redactor": "2.10.8",
"ether/seo": "3.7.4",
"ether/tags": "1.0.9",
"league/oauth2-client": "^2.5",
"league/oauth2-facebook": "^2.0",
"league/oauth2-google": "^3.0",
"league/oauth2-linkedin": "^5.1",
"samhernandez/craft-shortcodes": "1.1.0",
"sebastianlenz/linkfield": "1.0.25",
"solspace/craft-freeform": "3.13.13",
"verbb/navigation": "1.4.26",
"verbb/super-table": "2.7.2"

**Comment by brandonkelly** [comment, marked as answer]
You should not manually edit anything within `vendor/`.

If you are hoping to give your front-end a custom “Set Password” template, that’s covered [here](https://craftcms.com/knowledge-base/front-end-user-accounts#reset-password-forms) in the [Front-End User Accounts](https://craftcms.com/knowledge-base/front-end-user-accounts) KB article.

**Reply by ManiakMill** [reply]
I understand this. I didn't try to change any files in vendor/ folder. My target is add some elements to this form without changing the cp routes. I read this manual and try to make this with config  'setPasswordPath' and nothing happened. Form is still the same. Another problem I add this parameter and checked the link in the user email to this form, didn't change too.

<img width="242" alt="image" src="https://user-images.githubusercontent.com/2777750/176022599-0a7bcfc2-37bb-4d7c-9eb7-7136985106ab.png">

<img width="533" alt="image" src="https://user-images.githubusercontent.com/2777750/176023119-d1cbb175-63a0-482b-af8e-c132ecb4943f.png">

[headlessMode](https://craftcms.com/docs/3.x/config/config-settings.html#headlessmode) is disabled

**Reply by brandonkelly** [reply]
`setPasswordPath` is only for front-end Set Password forms.

**Reply by ManiakMill** [reply]
Hello. Ty for your help I understand the problem.
